site/base.less

```diff
@@ -6,7 +6,7 @@ html, body {
   -ms-text-size-adjust: 100%;
   margin: 0;
   padding: 0;
-  -webkit-font-smoothing: antialiased;
+  -webkit-font-smoothing: subpixel-antialiased;
```

Comment:
I think it's worth filing an issue for it, but I don't think it relates to your changeset and alters the styling of the docs so I'd rather not block merge on time to research it.

If I'm missing something and it's related to the changeset though, let me know.